docs/general/getting-started.mdx

```diff
@@ -66,9 +66,9 @@ Forecast vs Budget, Budget vs Last Year Actuals etc. The data series is
 at a certain time grain. This would typically be 1 year by 12 months or
 a multiyear view like 5 years plan by year.
 
-The selecting of the time series will often be based on Filters or
+A selection of time series will often be based on Filters or
```

Comment:
we can keep it as "The selection of time series or Time Series selection"...

Reply:
Ok, so shall I go with this one - "The selection of time series.." ?

Reply:
Yes, we can go with this option. 👍